projects/angular/src/forms/styles/_mixins.forms.scss

```diff
@@ -205,17 +205,33 @@
 
 @mixin range-style(
   $height,
+  $height-prop: null,
   $background-color,
   $background-color-prop: null,
   $use-custom-props: variables.$clr-use-custom-properties
 ) {
   -webkit-appearance: none;
   left: 0;
-  height: $height;
 
+  @include mixins.css-var(height, $height-prop, $height, $use-custom-props);
   @include mixins.css-var(background-color, $background-color-prop, $background-color, $use-custom-props);
 }
 
+@mixin range-border-style(
+  $border-color,
```

Comment:
maybe rename to border-color-fallback, as well as the other fallbacks, as it's misleading